user-guide/Advanced_Modules/User-Defined_APIs/Defining_an_API/UD_APIs_Defining_a_new_API.md

```diff
@@ -99,6 +104,24 @@ There are two ways to pass data to the API script if you make use of the *OnApiT
 > [!TIP]
 > To see how these two options are used in an API script, see [API script examples](xref:UD_APIs_API_script_examples).
 
+##### Query parameters
+
+Query parameters are available in the QueryParameters property in the `ApiTriggerInput`. The `IQueryParameters` exposes following methods:
+
+| Method name | Return type | Summary |
+|-------------|-------------|---------|
+| `TryGetValues(string key, out List<string> values)` | `bool` | Tries to get the value(s) for the associated key. Returns `true` in case value(s) are found for that key, `false` if not. The values will be in the `out` parameter. Key is case-sensitive! |
```

Comment:
```suggestion
| `TryGetValues(string key, out List<string> values)` | `bool` | Tries to get the value(s) for the associated key. Returns `true` in case value(s) are found for that key, `false` if not. The values will be in the `out` parameter. The key is case-sensitive. |
```

Reply:
Above comment can be applied to the other rows here as well. Maybe we can remove it since it is already included as was note below?